web/templates/typeahead_list_item.hbs

```diff
@@ -17,6 +17,20 @@
     <i class="typeahead-image zulip-icon zulip-icon-triple-users no-presence-circle" aria-hidden="true"></i>
 {{/if}}
 {{!-- Separate container to ensure overflowing text remains in this container. --}}
+{{~#if streamData}}
```

Comment:
Structure this similar to the above options, like `is_person` or `is_user_group`, have an `is_stream_topic`, and follow similar code patterns that those do.